src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body class="page">
+    <div class="card" data-qa="card">
+      <img 
+       src="/styles/images/imac.jpeg" 
+       alt="image"
+       class="card__image">
+
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <h3 class="card__code">Product code: 195434</h3>
```

Comment:
```suggestion
      <p class="card__code">Product code: 195434</p>
```
it's not really a heading, so you can use paragraph tag `<p>` instead